src/concurrent_map/device/count_kernel.cuh

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 Saman Ashkiani
```

Comment:
change this to "Copyright 2019 University of California, Davis".
I should change the rest as well. I forgot to do it when I change the ownership of the repo.